Formula/p/pdfalyzer.rb

```diff
@@ -0,0 +1,87 @@
+class Pdfalyzer < Formula
+  include Language::Python::Virtualenv
+
+  desc "PDF analysis toolkit"
+  homepage "https://github.com/michelcrypt4d4mus/pdfalyzer"
+  url "https://files.pythonhosted.org/packages/e8/ff/54dd0e10f7de21eec27d670e37c06e321eb44d74217b2c2238baeab5a804/pdfalyzer-1.14.5.tar.gz"
```

Comment:
this did not get marked as latest release yet.

Reply:
They released 1.14.6. I've updated to that.